setup.cfg

```diff
@@ -1,2 +1,6 @@
 [metadata]
 license_file = LICENSE
+
+[flake8]
```

Comment:
This is something that I'm using for local linting, up for debate if these settings belong in the repo.  Doesn't seem like it would hurt to have it configured for people that use it.